stream-compositions/api/integrations-api/payment-order-integration-api/README.md

```diff
@@ -0,0 +1,25 @@
+# payment-order-integration-api
+
+> _Fill out this file with some information about your OpenAPI spec._
```

Comment:
We removed a readme from the other integration services already. I'm going to do the same with this one. We'll leave it to the custom implementation to document the details of this service since this will be used for different core integrations.